.ci/travis.sh

```diff
@@ -56,9 +56,10 @@ sevntu-checks)
   mvn -e -Pcoverall install
   mvn -e verify -Pno-validations,selftesting
   mvn -e javadoc:javadoc
-  if [[ $TRAVIS == 'true' ]]; then
-    mvn -e -Pcoverall jacoco:report coveralls:report
-  fi
+  # until problem coverall back to online
```

Comment:
How will we know it's back online?

Reply:
We do not use this service, it has fancy badge and html report, I do not remember when last time I used it.